bitnami/kafka/templates/kafka-metrics-deployment.yaml

```diff
@@ -5,7 +5,6 @@ metadata:
   name: {{ template "kafka.fullname" . }}-exporter
   labels: {{- include "kafka.labels" . | nindent 4 }}
     app.kubernetes.io/component: metrics
-  annotations: {{ include "kafka.tplValue" ( dict "value" .Values.metrics.kafka.annotations "context" $) | nindent 4 }}
```

Comment:
They're useless. These annotations are used by Prometheus to auto-discover pods that must be scraped. However, since we're already adding them on the `kafka-metrics-svc.yaml`, they're already being discovered.

Reply:
👍